pootle/apps/pootle_store/views.py

```diff
@@ -468,7 +491,11 @@ def get_response_data(self, context):
             'editor': self.render_edit_template(context),
             'tm_suggestions': self.object.get_tm_suggestions(),
             'is_obsolete': self.object.isobsolete(),
-            'sources': self.get_sources()}
+            'sources': self.get_sources(),  # TODO: replace with alt_sources (?)
```

Comment:
I understand you reasoning here.

Generally speaking part of me thinks we shouldn't bother much about mapping response data 1:1 to specific UI needs, since data can be reshaped to the needs once it's in the client. There should be some balance between convenience and efficiency. Note 1:1 mapping ends up requiring ad-hoc endpoints with REST (something like [GraphQL](http://graphql.org/) makes that a non-issue by the way) although I am aware these are internal endpoints and the API surface will be relatively small.